CesiumRasterOverlays/include/CesiumRasterOverlays/RasterOverlayUtilities.h

```diff
@@ -116,7 +116,9 @@ struct CESIUMRASTEROVERLAYS_API RasterOverlayUtilities {
       bool hasInvertedVCoordinate = false,
       const std::string_view& textureCoordinateAttributeBaseName =
           DEFAULT_TEXTURE_COORDINATE_BASE_NAME,
-      int32_t textureCoordinateIndex = 0);
+      int32_t textureCoordinateIndex = 0,
+      const CesiumGeospatial::Ellipsoid& ellipsoid =
+          CesiumGeospatial::Ellipsoid::WGS84);
```

Comment:
Use `CESIUM_DEFAULT_ELLIPSOID` here.

Reply:
Can't, it would cause a compile error because of the other default parameters before it.